modules/enabling-cluster-capabilites.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="enabling_the_cluster_capabilities_{context}"]
+= Enabling the cluster capabilities
+
+As a cluster administrator, you can enable the capabilities in the cluster by using OpenShift CLI (oc).
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the jq package.
+
+.Procedure
+
+. Add to `additionalEnabledCapabilities` or adjust `baselineCapabilitySet` by running the following command:
++
+[source,terminal]
+----
+$ oc get -o json clusterversion version | jq -c '.status.capabilities.enabledCapabilities' ["openshift-samples"]
+----
+
+. Append to marketplace by running the following command:
```

Comment:
@shellyyang1989  shouldn't this be "To set the `additionalEnabledCapabilities`, run the following command:" ?

Reply:
ah, right. Let's update it.

Reply:
done.